src/app/error/error.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable, ErrorHandler } from '@angular/core';
```

Comment:
TODO: should be a common ErrorService https://github.com/PRX/styleguide.prx.org/issues/52